Gemfile.lock

```diff
@@ -252,7 +260,7 @@ GEM
       nokogiri (~> 1.8)
 
 PLATFORMS
-  ruby
+  x86_64-linux
```

Comment:
?

Reply:
again, this is what was set when creating the .lock file